EMIP/1-1000/EMIP00125.xml

```diff
@@ -190,6 +191,99 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
                        </layout>
                      </layoutDesc>
                   </objectDesc>
+                  <handDesc>
+                    <handNote xml:id="h1" script="Ethiopic">
+                    <seg type="rubrication">
+                    <foreign xml:lang="gez">ኢግዚአብሔር፡</foreign>
+                    <note>In the first three works</note>
+                    <foreign xml:lang="gez">ማርያም፡</foreign>
+                    <note>In the final two works</note>
+                    </seg>
+                    </handNote>
+                    </handDesc>
+                  <decoDesc>
+                    <decoNote type="miniature" xml:id="d01"><locus target="#iv(erso)"/><desc><ref type="authFile" corresp="AT1035HolyMan"> <persName ref="PRS4585Georgeo"/> and the <term key="dragon"/> </ref></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d02"><locus target="#iiv(erso)"/><desc><ref type="authFile" corresp="AT1031VirginAndChild"></ref></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d03"><locus target="#iiir(ecto)"/><desc><ref type="authFile" corresp="AT1082Angel">, with processional <term key="cross"/> before crowd‎</ref></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d04"><locus target="#iiiv(erso)"/><desc><ref type="authFile" corresp ="AT1049Resurrection"/></desc></decoNote>
+                    <decoNote type="miniature" xml:id="d05"><locus target="#137v"/><desc><ref type="authFile" corresp ="AT1035HolyMan"/> with <term key="Bible"/>(?) before crowd </desc></decoNote>
+                    <decoNote type="miniature" xml:id="d06"><locus target="#138r"/><desc><ref type="authFile" corresp="AT1065RoadtoCalvary"></ref></desc></decoNote>
+                    <decoNote type="frame" xml:id="d07">
+                      Decorative designs:
+                      <locus target="#1r #35r #72r #108r #126r"/> (haräg);
+                      <locus target="#7r #13r #19v #28r #48v"/> (line of alternating red and black dots);
+                      <locus target="#34v"/> (multiple full stops);
+                      <locus target="#40r #59r #82v #93v #96r #102r #107v"/> (multiple full stops connected with alternating red and black dots);
+                      <locus target="#66v #119r #125v"/> (a line of alternating red and black dots and a line of full stops)
+                    </decoNote>
+                  </decoDesc>
+                  <additions>
+                    <list>
+                      <item xml:id="a01">
+                    <desc type="Excerpt">
+                      <locus target="#iir(ecto)"/>
+                      Portion of Psalms 115–117‎, <title type="complete" ref="LIT2000Mazmur"/>
```

Comment:
This <title/> should not be marked with @type=complete, especially since these are excerpts. 
the @ref can be made more precise, by pointing at LIT2000Mazmur#Ps115 etc.